packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -95,14 +96,20 @@ class CodeSizeController {
     // TODO(peterdjlee): Use user input data instead of hard coded data.
     changeSnapshotFile(pathToFile);
 
-    // Build a [Map] object containing heirarchical information for [inputJsonMap].
-    final processedJsonMap = treemapFromJson(_jsonForFile(pathToFile));
+    final json = _jsonForFile(pathToFile);
+    Map<String, dynamic> processedJson;
+    if (pathToFile.contains('app_size')) {
```

Comment:
instead of checking the path, can you check for the "apk" bit you added to the json? This check will still be valid once we use real data and not stub data.

Reply:
Done